module_utils/ansible_nailgun_cement.py

```diff
@@ -9,6 +9,7 @@
 from nailgun.entities import (
     AbstractComputeResource,
     AbstractContentViewFilter,
+    AuthSourceLDAP,
```

Comment:
Oh, and i assumed, `AuthSourceLDAP` was tested _after_ `ActivationKey`.
Can you put it back in alphabethical order?
It might fix your test case, but i do not like that it depends on the order...

Reply:
Absolutely. I'm also fixing the order in `test_crud.py`.